core/src/com/team3gdx/game/food/Ingredient.java

```diff
@@ -64,6 +71,8 @@ public Ingredient(Vector2 pos, float width, float height, String name, int ideal
 				false);
 	}
 
+	//overload constructor so we can define idealMixTime only if necessary
```

Comment:
Unnecessary and incorrect comment. The overloaded constructor only serves to make a copy, and neither incorporate `idealMixTime`.

Reply:
Still not addressed